tests/dummy/app/pods/demo/template.hbs

```diff
@@ -40,12 +40,13 @@ frost-modal testbed
       <div>Test Information block</div>
     </div>
   {{/block-slot}}
-  {{#block-slot slot 'footer' as |controls|}}
+  {{#block-slot slot 'footer' as |controls modalClose|}}
     {{controls.cancel}}
     {{controls.button
       text='Custom action'
       onClick=(action 'myCustomAction')}}
     {{controls.confirm
+      onClick=modalClose
```

Comment:
This would move back to the default (no need to specify the close) - closeOnClick=false would be available for the addons extending the base modal